src/modules/workflowsettings/Elsa.WorkflowSettings/Services/WorkflowSettings/WorkflowSettingsManager.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.WorkflowSettings.Abstractions.Providers;
+using Elsa.WorkflowSettings.Abstractions.Services.WorkflowSettings;
+using Elsa.WorkflowSettings.Models;
+
+namespace Elsa.WorkflowSettings.Services.WorkflowSettingsContexts
+{
+    public class WorkflowSettingsManager : IWorkflowSettingsManager
+    {
+        private readonly IEnumerable<IWorkflowSettingsProvider> _workflowSettingsProviders;
+
+        public WorkflowSettingsManager(IEnumerable<IWorkflowSettingsProvider> workflowSettingsProviders)
+        {
+            _workflowSettingsProviders = workflowSettingsProviders;
+        }
+
+        public async ValueTask<WorkflowSetting> LoadSettingAsync(string workflowBlueprintId, string key, CancellationToken cancellationToken = default)
+        {
+            var providers = _workflowSettingsProviders;
+
+            var value = new ValueTask<WorkflowSetting>(new WorkflowSetting());
```

Comment:
Since your method is marked `async`, you should not instantiate `ValueTask<>`. Just instantiate `WorkflowSetting` directly.